MobiusCore/Source/EffectHandlers/EffectHandler.swift

```diff
@@ -41,27 +42,27 @@ public protocol EffectHandler {
     /// - Parameter input: The effect being handled
     /// - Parameter callback: The `EffectCallback` used to communicate with the associated Mobius loop.
     func handle(
-        _ input: Effect,
+        _ input: EffectParameters,
```

Comment:
How about naming the argument `effectParameters` or something like that as well?